src/components/AuthenticationForm/AuthenticationForm.tsx

```diff
@@ -169,7 +143,7 @@ export default function AuthenticationForm({ screen }: Props) {
               error={inputErrors[field.id]}
               required
               handleChange={onChangeHandler}
-              handleBlur={onCheckRules}
+              handleBlur={() => []}
```

Comment:
If we are not going to use the onBlur event, we should remove it from the InputField props

Reply:
Sure. I kept it since the other components still have it and I wanted to avoid merge conflicts with the other branches. But it will be removed